lib/routes/swagger.js

```diff
@@ -43,6 +43,71 @@ module.exports = function (app) {
    *   Locale:
    *     type: string
    *     minimum: 1
+   *
+   * parameters:
+   *   Pkg:
+   *     in: path
+   *     name: pkg
+   *     required: true
+   *     schema:
+   *       $ref: '#/definitions/PackageName'
+   *     description: The package name
+   *   Env:
+   *     in: path
+   *     name: env
+   *     required: true
+   *     schema:
+   *       $ref: '#/definitions/Environment'
+   *     description: The environment
+   *   Version:
+   *     in: path
+   *     name: version
+   *     required: true
+   *     schema:
+   *       $ref: '#/definitions/VersionNumber'
+   *     description: The package version
+   *
+   * content:
+   *   Error:
+   *     application/json:
+   *       schema:
+   *         type: object
+   *         properties:
+   *           code:
+   *             type: string
+   *           message:
+   *             type: string
+   * responses:
+   *   Standard:
+   *     200:
+   *       description: OK
+   *     400:
+   *       $ref: '#/responses/Standard400'
+   *     403:
+   *       $ref: '#/responses/Standard403'
+   *     404:
+   *       $ref: '#/responses/Standard404'
+   *     500:
+   *       $ref: '#/responses/Standard500'
+   *   Standard304:
```

Comment:
this should be 204 right?

Reply:
304. https://github.com/godaddy/warehouse.ai/blob/master/lib/npm/routes.js#L272

Reply:
There are some 204s but I left them in place. I could have doc'd the 304 in place as well and probably should have.

Reply:
@msluther I was being dumb and forgot 304 is for the caching bit and not an awkward redirect code